examples/lib/stories/camera_and_viewport/camera_component_properties_example.dart

```diff
@@ -41,6 +44,19 @@ class CameraComponentPropertiesExample extends FlameGame {
     _camera?.viewport.size = size * 0.7;
     _camera?.viewport.position = size * 0.6;
   }
+
+  @override
+  // ignore: must_call_super
+  void onTapDown(int pointerId, TapDownInfo info) {
+    final canvasPoint = info.eventPosition.widget;
+    for (final cp in componentsAtPoint(canvasPoint)) {
```

Comment:
```suggestion
    for (final componentPoint in componentsAtPoint(canvasPoint)) {
```

Reply:
I had this at first, but then it creates such abominations as `componentPoint.component` and `componentPoint.point` below, so I figured that an abbreviation might work better here.

Reply:
I think that is fine, it'll be easier to read especially since this is in an example.
(But this also shows why I don't like that tuple).

Reply:
Maybe we just need a better name for it? Otherwise, it's a pretty standard implementation of a `Pair` class.

Reply:
Yeah, possibly... Do you have any more names in mind?

Reply:
Did you come up with any other names? I forgot to think more about it.

Reply:
Maybe `queryComponentsAtPoint()`?

Reply:
I don't think that improves it, let's keep it like it is, I really can't come up with anything either.